xee/ext_integration_test.py

```diff
@@ -358,6 +358,43 @@ def test_honors_projection(self):
     self.assertEqual(ds.dims, {'time': 4248, 'lon': 3600, 'lat': 1800})
     self.assertNotEqual(ds.dims, standard_ds.dims)
 
+  @absltest.skipIf(_SKIP_RASTERIO_TESTS, 'rioxarray module not loaded')
+  def test_expected_double_precision_transform(self):
+    data = np.empty((162, 120), dtype=np.float32)
+    # An example of a double precision bbox
+    bbox = (
+        -53.94158617595226,
+        -12.078281822698678,
+        -53.67209159071253,
+        -11.714464132625046,
+    )
+    x_res = (bbox[2] - bbox[0]) / data.shape[1]
+    y_res = (bbox[3] - bbox[1]) / data.shape[0]
+    raster = xr.DataArray(
+        data,
+        coords={
+            'y': np.linspace(bbox[3], bbox[1] + x_res, data.shape[0]),
+            'x': np.linspace(bbox[0], bbox[2] - y_res, data.shape[1]),
+        },
+        dims=('y', 'x'),
+    )
+    ic = (
+        ee.ImageCollection('UCSB-CHG/CHIRPS/DAILY')
+        .filterDate(ee.DateRange('2014-01-01', '2014-01-02'))
+        .select('precipitation')
+    )
+    xee_dataset = xr.open_dataset(
+        ee.ImageCollection(ic),
+        engine='ee',
+        geometry=tuple(raster.rio.bounds()),
+        scale=raster.rio.resolution()[0],
```

Comment:
Related: should we let users pass in x and y scale?

Reply:
(Is this possible somehow today, eg via proj?)

Reply:
Yes, it is. Great call! I added this to the integration test in https://github.com/google/Xee/pull/138/commits/a2b3c55723a26f15c4156911773ab8b50aaad5b4